build.gradle

```diff
@@ -80,6 +87,11 @@ shadowJar {
     // Removes the 'All' in Classifier to have downloadable artifact by default.
     archiveClassifier.set(null)
 
+    duplicatesStrategy(DuplicatesStrategy.EXCLUDE)
```

Comment:
When this is in place and I try to use the .jar, while jmeter 5.6.2 is loading I get following error message:
```
An error occurred: Cannot invoke "java.util.jar.Manifest.getMainAttributes()" because the return value of "java.util.jar.JarFile.getManifest()" is null
```
When I removed the line, JMeter loaded successfully.